examples/combobox/combobox-autocomplete-list.html

```diff
@@ -53,7 +53,7 @@ <h2 id="ex_label">Example</h2>
           <div class="group">
             <input id="cb1-input" class="cb_edit" type="text" role="combobox" aria-autocomplete="list"
               aria-expanded="false" aria-controls="cb1-listbox">
-            <button id="cb1-button" tabindex="-1" aria-label="Open">
+            <button id="cb1-button" tabindex="-1"  aria-label="State" aria-expanded="false" aria-controls="cb1-listbox">
```

Comment:
Same recommendation as above:

```suggestion
            <button id="cb1-button" tabindex="-1"  aria-label="States" aria-expanded="false" aria-controls="cb1-listbox">
```

Reply:
Fixed